app/controllers/v1/workflow/processes_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::Workflow::ProcessesController < ApiController
   def index
-    # find the current_user's workflow, or load by :workflow_id
-    workflow = Workflow::Instance::Workflow.find_by!(external_identifier: params[:workflow_id])
+    query = params[:workflow_id] ? { external_identifier: params[:workflow_id] } : { id: workflow_id }
+    workflow = Workflow::Instance::Workflow.find_by!(query)
```

Comment:
@keithtom this endpoint is `/v1/workflow/workflows/:workflow_id/processes`. Do we want the optionality of finding the current user's workflow or always passing in the id?

Reply:
It is a good question.  I think we can keep this here.
I'm somewhat betting down the road we'll have other non-ssj workflows for a single user and this endpoint will be useful in surfacing multiple workflows.